src/main/java/in/projecteka/consentmanager/link/hip_link/model/request/AuthInitRequest.java

```diff
@@ -0,0 +1,13 @@
+package in.projecteka.consentmanager.link.hip_link.model.request;
+
+import lombok.Value;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Value
+public class AuthInitRequest {
+    UUID requestId;
```

Comment:
This is not UUID but a string. We are advising the UUID format, but need not be so, its string in the contract.

Reply:
but requestIds are treated as UUID only, even in gateway, 

are you saying only for this or everything?

Reply:
I think we should do that everywhere